resources/ru/cemu.po

```diff
@@ -1,146 +1,1140 @@
+
+#: GameProfileWindow.cpp
+msgid ""
+"EXPERT OPTION\n"
+"This option will load libraries from the cafeLibs directory"
+msgstr ""
+"EXPERT OPTION\n"
+"Эта опция загрузит библиотеки из каталога cafeLibs"
+
+# Не совсем корректный перевод, однако передаёт суть происходящего
+#: GameProfileWindow.cpp
+msgid "Launch with gamepad view"
+msgstr "Запуск с окном геймпада"
+
+# Аналогично, не совсем корректно.
+#: GameProfileWindow.cpp
+msgid ""
+"Games will be launched with gamepad view toggled as default. The view can be "
+"toggled with CTRL + TAB"
+msgstr ""
+"Игры будут запущены с включенным по умолчанию окном геймпада. Вид можно "
+"переключить с помощью CTRL + TAB"
+
+#: GameProfileWindow.cpp
+msgid "CPU"
+msgstr "CPU"
+
+#: GameProfileWindow.cpp
+msgid "Mode"
+msgstr "Режим"
+
+#: GameProfileWindow.cpp
+msgid "Singlecore-Interpreter"
+msgstr "Одноядерный интепретатор"
+
+#: GameProfileWindow.cpp
+msgid "Singlecore-Recompiler"
+msgstr "Одноядерный рекомпилятор"
+
+#: GameProfileWindow.cpp
+msgid "Dualcore-Recompiler"
+msgstr "Двухъядерный рекомпилятор"
+
+#: GameProfileWindow.cpp
+msgid "Triplecore-Recompiler"
+msgstr "Трёхъядерный рекомпилятор"
+
+#: GameProfileWindow.cpp
+msgid "Set the CPU emulation mode"
+msgstr "Выберите режим эмуляции CPU"
+
+#: GameProfileWindow.cpp
+msgid "Thread quantum"
+msgstr "Thread quantum"
+
+# thread slice runtime?
+#: GameProfileWindow.cpp
+msgid ""
+"EXPERT OPTION\n"
+"Set the maximum thread slice runtime (in virtual cycles)"
+msgstr ""
+"EXPERT OPTION\n"
+"Установить максимальное время выполнения среза потока (в виртуальных циклах)."
+
+#: GameProfileWindow.cpp
+msgid "cycles"
+msgstr "циклы"
+
+#: GameProfileWindow.cpp
+msgid "Extended texture readback"
+msgstr "Расширенное считывание текстур"
+
+#: GameProfileWindow.cpp
+msgid ""
+"Improves emulation accuracy of CPU to GPU memory access at the cost of "
+"performance. Required for some games."
+msgstr ""
+"Улучшает точность эмуляции доступа CPU к памяти GPU ценой "
+"производительности. Требуется для некоторых игр."
+
+#: GameProfileWindow.cpp GeneralSettings2.cpp
+msgid "Precompiled shaders"
+msgstr "Предварительно скомпилированные шейдеры"
+
+#: GameProfileWindow.cpp GeneralSettings2.cpp
+msgid "auto"
+msgstr "авто"
+
+#: GameProfileWindow.cpp GeneralSettings2.cpp
+msgid "enable"
+msgstr "включено"
+
+#: GameProfileWindow.cpp GeneralSettings2.cpp
+msgid "disable"
+msgstr "отключено"
+
+#: GameProfileWindow.cpp GeneralSettings2.cpp
+msgid ""
+"Precompiled shaders can speed up the load time on the shader loading "
+"screen.\n"
+"Auto will enable it for AMD/Intel but disable it for NVIDIA GPUs as a "
+"workaround for a driver bug.\n"
+"\n"
+"Recommended: Auto"
+msgstr ""
+"Предварительно скомпилированные шейдеры могут ускорить время начальной "
+"загрузки шейдеров.\n"
+"Auto включит его для AMD/Intel, но отключит его для NVIDIA GPU из-за ошибки "
+"драйвера.\n"
+"\n"
+"Рекомендуется: Авто"
+
+#: GameProfileWindow.cpp
+msgid "Shader mul accuracy"
```

Comment:
"mul" here is used as a shorthand for "multiplication". This string should be translated.